### PR TITLE
Add getting ChakraCore binaries

### PR DESCRIPTION
**Pull request by liminzhu**
Add a wiki on how to grab CC binaries.

### CODE REVIEW

**Comment by liminzhu** [comment]
@obastemur @dilijev

File: Building-ChakraCore.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Visit [[Getting ChakraCore binaies]] if you intend to just get the ChakraCore binaries.
```
Comment (dilijev) [inline Building-ChakraCore.md]: perhaps: "if you would simply like to download pre-compiled binaries."

File: Getting-ChakraCore-binaries.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Instead of [[Building ChakraCore]], you can get ChakraCore binaries through the [release](https://github.com/Microsoft/ChakraCore/releases) page or the following channels.
```
Comment (dilijev) [inline Getting-ChakraCore-binaries.md]: change the last `.` to `:`

File: Getting-ChakraCore-binaries.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+ 
+* For Windows binaries, use [Nuget/MyGet feed](https://github.com/Microsoft/ChakraCore/wiki/NuGet-Packages).
+* For Linux/macOS binaries, run `curl -SL https://aka.ms/chakracore/install | bash` in terminal to download the latest ChakraCore release package hosted on Azure.
+
+### Third party tools (not maintained by the ChakraCore team)
+* On macOS, run `brew install chakra` to install the latest ChakraCore release package.
```
Comment (dilijev) [inline Getting-ChakraCore-binaries.md]: nit: `macOS` -- consistency of terminology, don't we usually write `OSX`?

File: Getting-ChakraCore-binaries.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+Let us know if you want to add or already added ChakraCore to another package manager.
```
Comment (dilijev) [inline Getting-ChakraCore-binaries.md]: > want to add or already added

`want to add or have already added`

**Review by dilijev — APPROVED** [review]
Minor grammar nits, but LGTM

**Comment by liminzhu** [comment]
@dilijev shall we merge this one before https://github.com/Microsoft/ChakraCore/pull/2960?

**Comment by dilijev** [comment]
@liminzhu SGTM -- then I'll update that to point to the new page.

**Pull request merged**